include/APIClient/GPURequestor.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "restclient-cpp/restclient.h"
+#include "restclient-cpp/connection.h"
+
+#if defined(LUG_SYSTEM_WINDOWS)
+#pragma warning(push)
+#pragma warning(disable : 4003)
+#endif
+#include <json/json.hpp>
+#if defined(LUG_SYSTEM_WINDOWS)
+#pragma warning(pop)
+#endif
+
+#include "Router.hpp"
+
+using JSONResponse = std::tuple<int, nlohmann::json>;
+using ResponseCode = int;
+
+namespace APIClient {
+class GPURequestor {
```

Comment:
Isn't it better to use a non-instantiable class?
See the commentary about non-instantiable class in the file `Router.cpp` for the arguments.